src/components/payment/FirstSignUp/FirstSignUp.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
its ok to put FirstSignUP js and css in payment folder. put those files in payment folder